scripts/pi-hole/js/settings.js

```diff
@@ -121,7 +121,8 @@ $(".confirm-disablelogging-noflush").confirm({
 });
 
 $(".api-token").confirm({
-  text: "Make sure that nobody else can scan this code around you. They will have full access to the API without having to know the password. Note that the generation of the QR code will take some time.",
+  text:
+    "Make sure that nobody else can scan this code around you. They will have full access to the API without having to know the password. Note that the generation of the QR code will take some time.",
```

Comment:
It is very likely this change

Reply:
Strange... this is was `npm run prettier:fix` introduced.

Thanks.

Reply:
Your `prettier` may be outdated, try `npm i` to upgrade it/install the latest version

Reply:
Thanks. Fixed